doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -0,0 +1,1016 @@
+.. _cartridge_kubernetes_guide:
+
+================================================================================
+Kubernetes guide
+================================================================================
+
+This guide covers the full life cycle of a Tarantool Cartridge app--from
+developing the app to operating it in Kubernetes.
+
+.. contents::
+
+.. _cartridge_kubernetes_installing_tools:
+
+--------------------------------------------------------------------------------
+Installion tools
+--------------------------------------------------------------------------------
+
+The following tools are needed:
+
+1. **cartridge-cli** is a utility for managing Cartridge applications.
+   We need the version 2.3.0 or higher. Installation instructions are available
+   `here <https://github.com/tarantool/cartridge-cli#installation>`_.
+   If the installation is successful, the *cartridge* utility will be available
+   in the system.
+
+   .. code-block:: console
+
+      $ cartridge version
+      ---
+      Tarantool Cartridge CLI v2.3.0 linux/amd64 commit: 06a5dad
+
+2. **kubectl** is a Kubernetes cluster management tool. We need the
+   version 1.16 or higher. Installation instructions can be found
+   `here <https://kubernetes.io/ru/docs/tasks/tools/install-kubectl/>`_.
+
+   .. code-block:: console
+
+      $ kubectl version --client
+      ---
+      Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:36:53Z",     GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
+
+3. **helm** is a package manager for Kubernetes apps. We need the
+   version 3.3.x. Installation instructions can be found
```

Comment:
Current version is 3.4.2. Is 3.4.x okay or not?

Reply:
Version 3.4.x is possible, but the version above will most likely not work